---
title: Research
date: 2022-10-24

type: landing

sections:
  - block: markdown
    content:
      title: 3D organization and function of the genome
      text: |
        <div class="research-flex">
          <div class="research-text">
            The information stored in our genome is intertwined with its function, such that, when cells are submitted to specific sets of conditions, they may pass on to their progeny their functional state. Since DNA has been identified as a critical carrier of genetic information and since the same DNA can correspond to alternative, heritable functional states in certain cases, this transmission of cellular memory has been dubbed epigenetic inheritance. In the most spectacular way, this extends to inheritance of a phenotypic trait into subsequent generation, a phenomenon for which Conrad H. Waddington provided evidence some sixty years ago and which is well documented in plants. However, to which extent epigenetic inheritance operates in animals is hotly debated. Chromatin and its higher-order organization are epigenetic components that play an essential role in genome regulation. Both the DNA molecule and the nucleosomal histones can be extensively modified in a way that impinges on gene expression and may be inherited as well as erased upon specific regulatory cues. Furthermore, chromatin fibers can be folded into yet higher-order structures and chromosomes are confined in discrete “territories”. <br>
            We and others have discovered that metazoan chromosomes share a modular organization of their chromatin in structures called “Physical domains” or “Topologically Associating Domains” (TADs). TADs can be defined as linear units of chromatin that fold as discrete three-dimensional (3D) structures tending to favor internal, rather than external, chromatin interactions. TADs are delimited by boundaries, which contain housekeeping genes and insulator sites. They are detected by methods such as Hi-C, which allows genome-wide identification of chromatin contacts, and they correspond to Chromosomal Domains (CDs), previously identified by microscopy. The investigation of chromatin landscapes in metazoa through genome-wide association studies proved to be a fruitful approach. Theoretically, a huge number of chromatin types based on different combinations of chromatin-associated marks would be possible but, in fact, every report basically recapitulated the presence of an active chromatin environment, sometimes further subdivided, and of three major types of repressive chromatin: a Polycomb-repressed environment, a null environment and a heterochromatic environment. Strikingly, TADs were found to overlap with linear chromatin domains, indicating that epigenomic labeling of chromosome domains is intimately linked to their 3D folding. <br>
            We are trying to understand the principles governing 3D folding of the genome, from establishment of chromatin loops to the generation of chromosome domains, compartments, territories and the establishment of interchromosomal interactions. We use Drosophila, but also mouse and human cells, and state of the art molecular, genomic, computational and imaging approaches, in order to reach an integrated understanding of these different levels of genome organization.
          </div>
          <div class="research-video">
            <div class="video-container">
              <iframe src="https://www.youtube.com/embed/Pl44JjA--2k?start=1" frameborder="0" allowfullscreen title="Cavalli Lab Research Video"></iframe>
            </div>
          </div>
        </div>
    design:
      spacing:
        padding: ['40px', '0', '80px', '0']
  - block: markdown
    content:
      title: |
        WaddingtonMemory: Deciphering the role of regulatory factors driving epigenetic inheritance of alternative chromatin states
      text: |
        Epigenetics, the study of molecules and mechanisms that perpetuate alternative gene activity states in the context of the same DNA sequence, is an exciting field with important epistemological and biomedical implications, but the molecular mechanisms underlying epigenetic inheritance are still little understood. Polycomb group proteins are pleiotropic chromatin components that have been suggested to be capable of driving epigenetic inheritance and their dysregulation leads to cell fate changes and is associated with cancer. Recently, we discovered that a transient decrease in expression of a Polycomb gene can drive the formation of tumors of epigenetic nature, i.e. in the absence of DNA mutations. The goal of WaddingtonMemory is to decipher how epigenetic components can lead to stable changes in cell fate. <br>
        Specifically, we will: <br>
        <strong style="display:inline">Aim 1:</strong> Identify the molecular steps leading to epigenetic cell fate derailment following transient Polycomb protein depletion in Drosophila. We will perform a time-course study using bulk and single-cell multiomic and imaging approaches in order to dissect the dynamics of cell fate transformation. <br>
        <strong style="display:inline">Aim 2:</strong> Identify the Polycomb-targets leading to cell fate dysregulation and decipher their mechanistic role. We will test candidate factors identified in Aim 1 in order to identify those that drive cell fate derailment and to elucidate their mode of action. <br>
        <strong style="display:inline">Aim 3:</strong> Test the role of epigenetic inheritance in mammalian cell differentiation. We will analyse the role of epigenetic inheritance in mouse gastruloids, an in vitro system that reflects cell differentiation events typically found in early embryogenesis. <br> 
        <strong style="display:inline">Together, this groundbreaking project will reveal how epigenetic components drive cell fate derailment and it will establish robust paradigms that can be utilized by the scientific community to discriminate between epigenetic inheritance and DNA sequence-mediated cell transformation.</strong>
        <p></p>
      design:
      spacing:
        padding: ['40px', '0', '80px', '0']
  - block: markdown
    content:
      title: |
        Mechanisms of Polycomb-mediated genome regulation
      text: |
        Polycomb group (PcG) and trithorax group (trxG) proteins are key regulators of the expression of major developmental genes. PcG proteins are able to silence gene expression, while trxG proteins counteract gene silencing in the appropriate cells. The current model proposes that a sequence-specific DNA binding protein called PHO binds at so-called Polycomb response elements (PREs). PHO might recruit the PcG complex called PRC2, which contains the core subunits E(z), a histone methyltransferase that trimethylates histone H3 lysine 27 (H3K27me3), Su(z)12, Esc and Nurf55. H3K27me3 might then be recognized by the chromo domain of the PC subunits of PRC1, which also contains Ph, PSC and Sce/dRing. Once recruited, PcG complexes can propagate silencing through cell division. Genome-wide mapping studies have shown that PcG target genes encode for components controlling major signalling pathways and, importantly, PcG misexpression has also been associated with many cancer types, including breast and prostate cancer. <br>
        In addition to their role in cellular memory, PcG proteins participate in dynamic gene regulatory processes. In flies, different cell lines have a partially different set of PcG bound sites and H3K27me3-marked genomic regions change during development. In mammalian embryonic stem cells, many PcG target genes have been reported to bear both repression- and activation-associated marks. Upon differentiation, these “bivalent states” are resolved into fully active or fully repressed. In some instances, PcG components may even activate transcription, although it is unclear whether this phenomenon is widespread or rare. Importantly, PcG proteins regulate the organization of their target genes in the three-dimensional space of the nucleus, and this regulatory function is involved in the maintenance of cellular memory. <br>
        We would like to understand the molecular mechanisms of action of these factors, the role of regulation of higher order chromatin structure and nuclear organization in gene regulation, and the key molecular pathways that are mobilized by these proteins to coordinate the regulation of cell differentiation with that of cell proliferation. In particular, our research aims at (1) understanding, on a genome-wide scale, how these proteins are targeted to DNA and what are the consequences of this targeting on chromatin structure (2) understanding the effect of PcG proteins on cell proliferation, cell differentiation and cell polarity, and to dissecting the key components regulated by PcG proteins to modulate these pathways in specific tissues and developmental processes; (3) identifying the rules governing the distribution of their target genes in the cell nucleus and the effect of this organization on gene expression.
      design:
      spacing:
        padding: ['40px', '0', '80px', '0']
  - block: markdown
    content:
      title: Methods used in our lab
      text: ""
    design:
      spacing:
        padding: ['0', '0', '10px', '0']
  - block: slider
    content:
      slides:
      - title: Super-resolution microscopy
        content: See related articles!
        align: right
        background:
          image:
            filename: SuperResolution_QS.jpg
            filters:
              brightness: 0.7
          position: right
          color: '#666'
        link:
          icon: graduation-cap
          icon_pack: fas
          text: Link
          url: ../contact/
    design:
      # Slide height is automatic unless you force a specific height (e.g. '400px')
      slide_height: ''
      is_fullscreen: true
      # Automatically transition through slides?
      loop: false
      # Duration of transition between slides (in ms)
      interval: 2000
---
